package-lock.json

```diff
@@ -262,11 +239,11 @@
     },
     "formatio": {
       "version": "1.2.0",
-      "resolved": "https://artifactory.secureserver.net/artifactory/api/npm/node-virt/formatio/-/formatio-1.2.0.tgz",
+      "resolved": "https://registry.npmjs.org/formatio/-/formatio-1.2.0.tgz",
```

Comment:
Probably a good idea to add a `.npmrc` file to this project pointing to `registry.npmjs.org` because there are a bunch of artifactory URLs in this shrinkwrap.

Reply:
Actually this repo always had this file. I believe this is an artifact of npm?

Reply:
You can also set `registry=` in your `.npmrc` to force npm to download from a specific registry. It's a good workaround to ensure you don't download from artifactory (and thus affect the shrinkwrap/lock file) when working on a public project.

Reply:
I would recommend we update this before merging.